build/bootstrap/refresh-autoshutdown.cron.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
It seems this file is not correctly named -- this is not a `cron` script and is not used as a `cron` job. It's a script that is run once on startup.

Reply:
The rationale of the name: `$action-$taget`. I can just rename it to something more generic, like "gceworker-startup-script.sh". WDYT?

Reply:
Works for me.